test/test_helper.rb

```diff
@@ -20,6 +20,7 @@
 
 # Do not allow network connections and external processes
 WebMock.disable_net_connect!(allow_localhost: true)
+WebMock.allow_net_connect!(net_http_connect_on_start: true)
```

Comment:
@lzap I think I should have rather used the `disable_net_connect!` for the `net_http_connect_on_star`t too

Reply:
I think it is correct, we should just add the option to the `disable_net_connect!`

Full disclosure, I still don't know what this does. It would be a great help for future if you could describe what issue we are trying to fix in the **commit message body** :)